wagtail/admin/templates/wagtailadmin/pages/listing/_list.html

```diff
@@ -1,79 +0,0 @@
-{% load i18n %}
-{% load l10n %}
-{% load wagtailadmin_tags %}
-<table class="listing {% if full_width %}full-width{% endif %} {% block table_classname %}{% endblock %}" {% if show_ordering_column %}aria-descrption="{% trans 'Press enter to select an item, use up and down arrows to move the item, press enter to complete the move or escape to cancel the current move.' %}"{% endif %}>
-    {% if show_ordering_column or show_bulk_actions %}
-        <col width="10px" />
-    {% endif %}
-    <col />
-    {% if show_parent %}
-        <col />
-    {% endif %}
-    <col width="12%" />
-    <col width="12%" />
-    <col width="12%" />
-    <col width="10%" />
-    <thead>
-        {% block post_parent_page_headers %}
-            {% include "wagtailadmin/pages/listing/_table_headers_explore.html" %}
-        {% endblock %}
-    </thead>
-    <tbody>
-        {% if pages %}
-            {% trans "Select page" as checkbox_aria_label %}
-            {% for page in pages %}
-                {% page_permissions page as page_perms %}
-                <tr {% if ordering == "ord" %}id="page_{{ page.id|unlocalize }}" data-page-title="{{ page.get_admin_display_title }}"{% endif %} class="{% if not page.live %}unpublished{% endif %} {% block page_row_classname %}{% endblock %}">
-                    {% if show_ordering_column %}
-                        <td class="ord">
-                            {% if orderable and ordering == "ord" %}
-                                <div class="handle" tabindex="0" aria-live="polite" data-order-handle>
-                                    {% icon name="grip" classname="default" %}
-                                    <span class="w-sr-only">
-                                        {% trans 'Drag' %}
-                                        <span data-order-label>Item {{ forloop.counter }} of {{ pages|length }}</span>
-                                    </span>
-                                </div>
-                            {% endif %}
-                        </td>
-                    {% elif show_bulk_actions %}
-                        {% include "wagtailadmin/bulk_actions/listing_checkbox_cell.html" with obj_type="page" obj=page aria_labelledby_prefix="page_" aria_labelledby=page.pk|unlocalize aria_labelledby_suffix="_title" %}
-                    {% endif %}
-                    <td id="page_{{ page.pk|unlocalize }}_title" class="title" valign="top" data-listing-page-title>
```

Comment:
It seems we need to re-instate the `valign="top"` via CSS, I remember seeing a commit that did this in some way but I can't find it anymore. Was that removed?

Reply:
85bf7ba49437bdd784376cbf68437f3de9b5bfbe updated _listing.scss to support setting the "align-top" classname on the `<td>` instead of the whole table; in df0ee5c7498c7ad50d5f5167a13442cc74846dbf we then pass that as the `classname` attribute on all the relevant columns (and the cell templates  all do `<td {% if column.classname %}class="{{ column.classname }}"{% endif %}>` as standard). Is that what you're looking for?

Reply:
Ah, found the issue: https://github.com/wagtail/wagtail/pull/10649#discussion_r1258578594